SU-SWS/ansible-playbooks
# SITES-732, SITES-843, SITES-858

# READY FOR REVIEW

# Summary
- Update acsf.aliases.drushrc.php to reflect `*.sites.stanford.edu` namespacing
- Update migration scripts so that people sites get sunetid.people.stanford.edu as domain, and Stack 3
- Replace all instances of cardinalsites.stanford.edu with sites.stanford.edu in migration scripts

# Needed By (Date)
- Soooon

# Criticality
- How critical is this PR on a 1-10 scale? 10/10
- The old scripts will be pretty broken once I refresh `test` and `dev`, because new sites no longer live in the cardinalsites.stanford.edu namespace.

# Steps to Test

1. Check out this branch
2. Migrate sites using the inventory that I am including at the bottom of this PR description
3. Ensure that everything migrates cleanly
4. Check sites for broken image links
5. Manually review changes to scripts for clarity and comment accuracy.

# Affected Projects or Products
- ACSF

# Associated Issues and/or People
- JIRA tickets
    -https://stanfordits.atlassian.net/browse/SITES-732
    -https://stanfordits.atlassian.net/browse/SITES-843
    -https://stanfordits.atlassian.net/browse/SITES-858

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

# Inventory
```
[people]
jbickar

[hsdo]
jumpstart-hcbss vhost="buddhiststudies"

[depts]
encumbrances-intranet

[groups]
sws-migration-testing vhost="sws-oddball-vhost"

[people:vars]
group_ids=656
ignore_updb_errors="TRUE"
php_candidates_consequence="continue"

[hsdo:vars]
group_ids=606
ignore_updb_errors="TRUE"
php_candidates_consequence="continue"

[products:children]
hsdo

[depts:vars]
group_ids=206
ignore_updb_errors="TRUE"
php_candidates_consequence="continue"

[groups:vars]
group_ids=206
ignore_updb_errors="TRUE"
php_candidates_consequence="continue"
```
